docs/libmamba-vs-classic.md

```diff
@@ -14,18 +21,27 @@ If the given solution is not fully satisfying, try to restrict your request a bi
 For example, if you run `conda install scipy` and do not get the latest version, try using a more explicit command: `conda install scipy=X.Y`.
 ```
 
-You should probably still use `classic` in the following scenarios:
-
-* Backwards compatibility is very important in your use case, and you need your environments solved in the same way they have been solved in the past years.
-  This is a bit more important in long-lived environments than new ones.
+You might still need `classic` here and there, specially if backwards compatibility is very important in your use case, and you need your environments solved in the same way they have been solved in the past years.
+This is a bit more important in long-lived environments than new ones.
```

Comment:
```suggestion
The classic solver could be important to you if:

- Backwards compatibility is important (i.e. environments must be solved exactly as they always have been)
- The intentional deviations from the `classic` solver (see below) are not acceptable and you prefer the old behavior

These reasons could be especially important if you continue to use long lived environments that were initially created with the `classic` solver.
```

Again, I think the bullet points convey the information more concisely. I also reworded the piece about long lived environments.